assets/translations/en-us.yaml

```diff
@@ -82,7 +82,9 @@ locale:
 nav:
   backToRancher: Cluster Manager
   clusterTools: Cluster Tools
-  kubeconfig: Download KubeConfig
+  kubeconfigDownload: Download KubeConfig
```

Comment:
Since we have may related here, I'd suggest a kubeconfig key with child keys download, copy and options